### PR TITLE
Updated Contributing .md file

### LINKED ISSUE #39: add CONTRIBUTING.md
add contributing.md to give more information about your project,how to set up and run it and welcome new contributors

### PR DESCRIPTION
**Pull request by mansijain980**
# Description

I have modified the contributing.md file to make it readable and efficient .

Fixes #39 

 
# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by mansijain980** [comment]
@himanshu007-creator this issue will count under gssoc'21 ?

**Comment by rohank2502** [comment]
> 
> 
> @himanshu007-creator this issue will count under gssoc'21 ?

Don't worry this will count

**Review by rohank2502 — CHANGES_REQUESTED** [review]
Line 29: Change Repo to Repository

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,32 +1,59 @@
...
+### Step 0 : Find an issue🤔
 
-- Take a look at the Existing Issues or create your **own** Issues!
+- Take a look at the Existing [Issues](https://github.com/himanshu007-creator/SFC-foundations/issues) or create your own Issues!
 - Wait for the Issue to be assigned to you after which you can start working on it.
 - Note : Every change in this project should/must have an associated issue.
 
 
-### Step 1 : Fork the Project
+### Step 1 : Fork the Repo😆
```
Comment (himanshu007-creator) [inline CONTRIBUTING.md:29]: replace the emoji with 🔗

**Review by himanshu007-creator — CHANGES_REQUESTED** [review]
And replace repo with repository as suggested by @rohank2502

**Pull request merged**